CIP-0048/README.md

```diff
@@ -0,0 +1,249 @@
+---
+CIP: 48?
+Title: 721 References and Payloads
+Authors: Jack (ada9000, https://forum.cardano.org/u/jack7e/, @ada9000_ twitter)
+Comments-URI: https://forum.cardano.org/t/cip-extended-721/
+Status: Draft
+Type: Informational
+Created: 2022-04-22 (updated 2022-09-22)
```

Comment:
```suggestion
Created: 2022-04-22
```
Version history strictly follows GitHub, so "updated" dates don't need to be in the header & our collation scripts can't process these because there's no standard form for them.  Current consensus is that particular dated versions can be identified using the GitHub commit hash (forming its own version policy).